x-pack/plugin/ccr/src/javaRestTest/java/org/elasticsearch/xpack/ccr/AutoFollowIT.java

```diff
@@ -373,28 +367,14 @@ public void testDataStreams() throws Exception {
             }
             assertBusy(() -> {
                 assertThat(getNumberOfSuccessfulFollowedIndices(), equalTo(initialNumberOfSuccessfulFollowedIndices + 3));
-                verifyDataStream(
-                    client(),
-                    dataStreamName,
-                    backingIndexName(dataStreamName, 1),
-                    backingIndexName(dataStreamName, 2),
-                    backingIndexName(dataStreamName, 3)
-                );
+                verifyDataStream(client(), dataStreamName, 1, 2, 3);
                 ensureYellow(dataStreamName);
                 verifyDocuments(client(), dataStreamName, numDocs + 2);
             });
 
         } finally {
-            cleanUpFollower(
-                List.of(backingIndexName(dataStreamName, 1), backingIndexName(dataStreamName, 2), backingIndexName(dataStreamName, 3)),
-                List.of(dataStreamName),
-                List.of(autoFollowPatternName)
-            );
-            cleanUpLeader(
-                List.of(backingIndexName(dataStreamName, 1), backingIndexName(dataStreamName, 2), backingIndexName(dataStreamName, 3)),
-                List.of(dataStreamName),
-                List.of()
-            );
+            cleanUpFollower(List.of(), List.of(dataStreamName), List.of(autoFollowPatternName));
+            cleanUpLeader(List.of(), List.of(dataStreamName), List.of());
```

Comment:
There are a few more places like this. When a data stream is deleted, all its backing indices are deleted as well, of course. Theoretically it's possible that there are backing indices that have a name that matches a data stream name (e.g. manually created or taken out of an existing data stream) but are not in the data stream, but that doesn't seem to happen in these tests, so we don't need to explicitly delete the backing indices.